src/basic_recipes/arrows.jl

```diff
@@ -21,31 +21,12 @@ grid.
 ## Attributes
 $(ATTRIBUTES)
 """
```

Comment:
In this PR I kept the docs in Makie.jl. 
The advantage is that the docstring extensions machinery is avaible in Makie but not in MakieCore.

The only observable difference is that before `arrows` and `arrows!` had exactly the same doc string. After this PR the `arrows` docstring stays the same, but `arrows!` docstring just says "see arrows".